base/libgit2/types.jl

```diff
@@ -576,6 +576,20 @@ Base.show(io::IO, ie::IndexEntry) = print(io, "IndexEntry($(string(ie.id)))")
     LibGit2.RebaseOptions
 
 Matches the `git_rebase_options` struct.
+
+The fields represent:
+  * `version`: version of the struct in use, in case this changes later. For now, always `1`.
+  * `quiet`: inform other git clients helping with/working on the rebase that the rebase
+    should be done "quietly". Used for interoperability. The default is `1`.
+  * `inmemory`: start an in-memory rebase. Callers working on the rebase can go through its
+    steps and commit any changes, but cannot rewind HEAD or update the repository. The
+    [`workdir`](@ref) will not be modified. Only present on libgit2 versions newer than 0.24.0.
```

Comment:
"New than" implies to me strictly greater than 0.24.0, which suggests that it was introduced in 0.24.1. Is that accurate?